tools/accuracy_checker/tests/test_segmentation_representation.py

```diff
@@ -0,0 +1,114 @@
+                assert np.array_equal(actual_arr.sort(axis=0), expected_arr.sort(axis=0))
+
+    def test_to_polygon_prediction(self):
+        predictions = make_segmentation_representation(np.array([[1, 0, 0, 0], [1, 1, 0, 0], [1, 1, 1, 0]]), False)
+        expected = {
+            0: [np.array([[1, 0], [3, 0], [3, 2]], dtype=np.int32)],
+            1: [np.array([[0, 0], [0, 2], [2, 2]], dtype=np.int32)]}
+        for prediction in predictions:
+            actual = prediction.to_polygon()
+
+        for key, _ in expected.items():
+            assert actual[key]
+            for actual_arr, expected_arr in zip(actual[key], expected[key]):
+                assert np.array_equal(actual_arr.sort(axis=0), expected_arr.sort(axis=0))
+
+    def test_to_polygon_prediction_with_argmax(self):
+        predictions = make_segmentation_representation(np.array(
+            [[[0.01, 0.99, 0.99, 0.99], [0.01, 0.01, 0.99, 0.99], [0.01, 0.01, 0.01, 0.99]],
+             [[0.99, 0.01, 0.01, 0.01], [0.99, 0.99, 0.01, 0.01], [0.99, 0.99, 0.99, 0.01]]]), False)
+        expected = {
+            0: [np.array([[1, 0], [3, 0], [3, 2]], dtype=np.int32)],
+            1: [np.array([[0, 0], [0, 2], [2, 2]], dtype=np.int32)]}
+        for prediction in predictions:
+            actual = prediction.to_polygon()
+
+        for key, _ in expected.items():
+            assert actual[key]
+            for actual_arr, expected_arr in zip(actual[key], expected[key]):
+                assert np.array_equal(actual_arr.sort(axis=0), expected_arr.sort(axis=0))
+
+class TestCoCoInstanceSegmentationRepresentation:
+
+    def test_to_polygon_annotation(self):
+        mask = [np.array([[1, 0, 0, 0], [1, 1, 0, 0], [1, 1, 1, 0]]),
+                np.array([[0, 1, 1, 1], [0, 0, 1, 1], [0, 0, 0, 1]])]
+        labels = [0, 1]
+        annotations = make_instance_segmentation_representation(mask, labels, True)
+        expected = {
+            0: [np.array([[1, 0], [3, 0], [3, 2]], dtype=np.int32)],
+            1: [np.array([[0, 0], [0, 2], [2, 2]], dtype=np.int32)]}
+        for annotation in annotations:
+            actual = annotation.to_polygon()
+
+        for key, _ in expected.items():
+            assert actual[key]
+            for actual_arr, expected_arr in zip(actual[key], expected[key]):
+                assert np.array_equal(actual_arr.sort(axis=0), expected_arr.sort(axis=0))
+
+    def test_to_polygon_prediction(self):
+        mask = [np.array([[1, 0, 0, 0], [1, 1, 0, 0], [1, 1, 1, 0]]),
+                np.array([[0, 1, 1, 1], [0, 0, 1, 1], [0, 0, 0, 1]])]
+        labels = [0, 1]
+        predictions = make_instance_segmentation_representation(mask, labels, False)
+        expected = {
+            0: [np.array([[1, 0], [3, 0], [3, 2]], dtype=np.int32)],
+            1: [np.array([[0, 0], [0, 2], [2, 2]], dtype=np.int32)]}
+        for prediction in predictions:
+            actual = prediction.to_polygon()
+
+        for key, _ in expected.items():
```

Comment:
why you need items() if you use only keys?

Reply:
replace to keys